tests/sync-test.js

```diff
@@ -85,6 +87,98 @@ describe("messaging between master and client", () => {
+  })
+
+  it('ensure clients can share a master', async () => {
+    // return
+    var {world1, masterWorld, client1, client2, master} = state;    
+    var testChannel = new L2LChannel(client1, "receiveOpsFromMaster", master, "receiveOpsFromClient")    
+    window.testChannel = testChannel
+    expect((testChannel.senderRecvrA.l2lclient) &&  (testChannel.senderRecvrA.l2lclient instanceof L2LClient)).equals(true,'client A not L2LClient')
+    expect((testChannel.senderRecvrB.l2lclient) &&  (testChannel.senderRecvrB.l2lclient instanceof L2LClient)).equals(true,'client B not L2LClient')
+
+    await testChannel.senderRecvrA.l2lclient.whenRegistered(300)
+    await testChannel.senderRecvrB.l2lclient.whenRegistered(300)
+
+    var channelId = testChannel.senderRecvrB.l2lclient.socketId.split('#')[1]
+
+    var testChannel2 = new L2LChannel(client2, "receiveOpsFromMaster", master, "receiveOpsFromClient")    
+    window.testChannel2 = testChannel2
+    expect((testChannel2.senderRecvrA.l2lclient) &&  (testChannel2.senderRecvrA.l2lclient instanceof L2LClient)).equals(true,'client A not L2LClient')
+    expect((testChannel2.senderRecvrB.l2lclient) &&  (testChannel2.senderRecvrB.l2lclient instanceof L2LClient)).equals(true,'client B not L2LClient')
+
+    await testChannel2.senderRecvrA.l2lclient.whenRegistered(300)
+    await testChannel2.senderRecvrB.l2lclient.whenRegistered(300)
+
+    expect(testChannel2.senderRecvrB === testChannel.senderRecvrB).equals(true,'Clients have different masters')
+    
+    testChannel.goOffline();
+    testChannel2.goOffline();
+  })
+
+  it('ensure clients can communicate across channel', async () => {
+    var {world1, masterWorld, client1, master} = state;  
+    var client1Buffer = []
+    var masterBuffer = []
+    var payload = "Test for lively.sync messaging"
+    var ack = 'received'
+    function masterCheck(item){      
+      masterBuffer.push(item)
+    }
+    function ackFn(item){
+        client1Buffer.push(item)        
+        client1.l2lclient.sendTo(master.l2lclient.id,'lively.sync',{payload: ack})
+    }
+    client1.receiveOpsFromMaster = ackFn;
+    master.receiveOpsFromClient = masterCheck;
+    
+    var testChannel = new L2LChannel(client1, "receiveOpsFromMaster", master, "receiveOpsFromClient")
+    await testChannel.senderRecvrA.l2lclient.whenRegistered(300)
+    await testChannel.senderRecvrB.l2lclient.whenRegistered(300)
+    
+    // testChannel.registerReceive(client1,client1.receiveOpsFromMaster)
+    // testChannel.registerReceive(master,master.receiveOpsFromClient)
+    master.l2lclient.sendTo(client1.l2lclient.id,'lively.sync',{payload: payload})
+    await promise.waitFor(200, () => client1Buffer.length  >= 1);
+    await promise.waitFor(200, () => masterBuffer.length >= 1);
+    testChannel.goOffline();
+    expect(client1Buffer[0].payload).equals(payload,'payload not correct')
+    expect(masterBuffer[0].payload).equals(ack,'payload not correct')
+  })
+
+})
```

Comment:
None of those tests actually runs lively.sync. How would the tests ""messaging between master and client - single op" and "syncing master with two clients - simple prop" look like when using l2l?